sparkling-water-mojo-scorer/example.py

```diff
@@ -0,0 +1,49 @@
+from pyspark.sql import SparkSession
+import argparse
+
+from pyspark.sql.types import StringType,FloatType
+from pyspark.sql.functions import monotonically_increasing_id
+
+def main():
+    spark = SparkSession.builder.appName('MOJO Pipeline Transformer').getOrCreate()
+    spark.sparkContext.setLogLevel('WARN')
+
+    parser = argparse.ArgumentParser()
```

Comment:
clarity: Consider moving this into a dedicated function, e.g., `_parse_args` to let `main` focus on the important stuff.
Also please consider keeping line length under 80 (or at least 100). It makes github PR diff much more readable :+1:.

Reply:
agree. Alternatively, use `Click` module. Its a bit more legible and slightly easier to use imho